src/noun.substance

```diff
@@ -5485,6 +5503,18 @@ w: tufa
 d: a soft porous rock consisting of calcium carbonate deposited from springs rich in lime
 hypo: rock
 
+w: calcareous_marlstone
+d: A mixed carbonate-clastic rock in which the clastic portion is restricted to clay-sized components. Consists of 25-50% clay and 50-75% calcite (Vinx, 2005; Füchtbauer, 1988)
```

Comment:
references are not part of the original WN. It would be better have a link to the Concept in the Ontology that contains the reference. Not include it in the glosses.

Reply:
Ok

Reply:
References removed